client/landing/gutenboarding/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum Steps {
+	IntentGathering = 0,
```

Comment:
Isn't it `0` by default? I was expecting an enum to be explicitly initialized when using something like `1` or `a`. But its just personal opinion and we can use the current approach as convention.

Reply:
You're right, implicit should be sufficient here :+1: